src/fabric/brick.rs

```diff
@@ -61,14 +57,19 @@ impl Fabric {
                     let ideal = self.ideal(alpha_index, omega_index, Baked::TARGET_FACE_STRAIN);
                     self.create_interval(alpha_index, omega_index, Link::pull(ideal))
                 });
-                // TODO: make sure there's only one face left
-                self.create_face(aliases, scale, spin, radial_intervals)
+                let single_alias: Vec<_> = aliases
+                    .into_iter()
+                    .filter(|alias| search_alias.matches(alias))
+                    .collect();
+                assert_eq!(single_alias.len(), 1, "filter must leave exactly one face alias");
+                self.create_face(single_alias, scale, spin, radial_intervals)
             })
             .collect();
         if let Some(id) = face_id {
+            let search_base = search_alias.with_base();
```

Comment:
```suggestion
            let search_alias = search_alias.with_base();
```
search_base is a more unclear name to me, so just shadow "search_alias"